vertical-pod-autoscaler/recommender/output/checkpoint_writer.go

```diff
@@ -75,3 +76,26 @@ func (writer *checkpointWriter) StoreCheckpoints() {
 		}
 	}
 }
+
+func buildAggregateContainerStateMap(vpa *model.Vpa, now time.Time) map[string]*model.AggregateContainerState {
```

Comment:
I'd add some info why this function is introduced (compared to original model.BuildAggregateContainerStateMap)

E.g. change name to buildCheckpointAggregateContainerStateMap